lib/node_modules/@stdlib/stats/base/dnanvarianceyc/src/addon.c

```diff
@@ -1,7 +1,7 @@
 /**
 * @license Apache-2.0
 *
-* Copyright (c) 2025 The Stdlib Authors.
+* Copyright (c) 2024 The Stdlib Authors.
```

Comment:
```suggestion
* Copyright (c) 2025 The Stdlib Authors.
```
no need for changes here